var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -61,7 +61,8 @@ def install(self, spec, prefix):
         autoconf()
 
         config_args = [
-            '--prefix={0}'.format(prefix)
+            '--prefix={0}'.format(prefix),
+            '--enable-shared'
```

Comment:
It might be a good idea to make sure every version of libint has this flag available. Version 1.x.x and version 2.x.x had very different options.